testcontainers/src/runners/sync_runner.rs

```diff
@@ -118,6 +119,20 @@ mod tests {
         runtime().block_on(client.network_exists(name)).unwrap()
     }
 
+    fn get_server_container(msg: Option<WaitFor>) -> GenericImage {
```

Comment:
This is always called with `None`. I think since this returns `GenericImage` adding the `WaitFor` can be done afterwards anyways.

Reply:
I've just followed the same pattern already existing. But you are right, the parameter is never used and I've thus removed it now